pig/src/main/java/org/jboss/pnc/bacon/pig/impl/repo/RepoManager.java

```diff
@@ -65,6 +78,7 @@
     private final boolean removeGeneratedM2Dups;
     private final Path configurationDirectory;
     private final boolean strictLicenseCheck;
+    private final boolean isTestMode;
```

Comment:
would be great to have it on WireMock that serves fake redhat artifacts, then we wouldn't need `istTestMode`, the test would be faster, and would also test `removeCommunityArtifacts` and `removeIrrelevantFiles`

Reply:
This would require more work to analyse all the wiremock calls 
I or some other person could use the wiremock later I think